src/Nethermind/Nethermind.Runner/configs/op-mainnet.json

```diff
@@ -1,12 +1,10 @@
 {
   "$schema": "https://raw.githubusercontent.com/NethermindEth/core-scripts/refs/heads/main/schemas/config.json",
   "Init": {
-    "ChainSpecPath": "chainspec/op-mainnet.json",
-    "GenesisHash": "0x7ca38a1916c42007829c55e69d3e9a73265554b586a499015373241b8a3fa48b",
+    "ChainSpecPath": "chainspec/op-mainnet.json.zst",
+    "GenesisHash": "0xdbf6a80fef073de06add9b0d14026d6e5a86c85f6d102c36d3d8e9cf89c2afd3",
```

Comment:
Why is it changed? Did you try to run archive and snap? Maybe it will brake something

Reply:
It was changed since we're pulling the data from the Superchain registry

https://github.com/ethereum-optimism/superchain-registry/blob/8ff62ada16e14dd59d0fb94ffb47761c7fa96e01/superchain/configs/mainnet/op.toml#L34

Now, this is wrong: Nethermind is not able to start due to a hash mismatch (the previous one was correct). I'm not sure what to make out of this. We already have some special logic for OP-mainnet but it should not be necessary to hardcode this hash.